app/assets/javascripts/discourse/app/components/sidebar/edit-navigation-menu/categories-modal.gjs

```diff
@@ -307,60 +408,82 @@ export default class SidebarEditNavigationMenuCategoriesModal extends Component
+                        }}
+                        class="sidebar-categories-form__category-label"
+                      >
+                        <div class="sidebar-categories-form__category-wrapper">
+                          <div class="sidebar-categories-form__category-badge">
+                            {{categoryBadge category}}
+
+                          </div>
+
+                          {{#unless category.parentCategory}}
+                            <div
+                              class="sidebar-categories-form__category-description"
+                            >
+                              {{dirSpan
+                                category.description_excerpt
+                                htmlSafe="true"
+                              }}
+                            </div>
+                          {{/unless}}
+                        </div>
 
-                      {{#unless category.parentCategory}}
-                        <div
-                          class="sidebar-categories-form__category-description"
-                        >
-                          {{dirSpan
-                            category.description_excerpt
-                            htmlSafe="true"
+                        <input
+                          {{on "click" (fn this.toggleCategory category.id)}}
+                          type="checkbox"
+                          checked={{has
+                            this.selectedCategoryIds
+                            category.id
                           }}
-                        </div>
-                      {{/unless}}
+                          id={{concat
+                            "sidebar-categories-form__input--"
+                            category.id
+                          }}
+                          class="sidebar-categories-form__input"
+                        />
+                      </label>
                     </div>
-
-                    <input
-                      {{on "click" (fn this.toggleCategory category.id)}}
-                      type="checkbox"
-                      checked={{has
-                        this.selectedSidebarCategoryIds
-                        category.id
-                      }}
-                      disabled={{not
-                        (includes this.filteredCategoryIds category.id)
-                      }}
-                      id={{concat
-                        "sidebar-categories-form__input--"
-                        category.id
-                      }}
-                      class="sidebar-categories-form__input"
-                    />
-                  </label>
-                </div>
+                  {{/with}}
+                {{else}}
+                  <div
+                    {{didInsert this.didInsert}}
+                    data-category-level={{c.level}}
+                    class="sidebar-categories-form__category-row"
+                  >
+                    <label
```

Comment:
I'm not sure this label element is needed for the "Show more" button (in the subcategory row, this ensure clicking the row toggles the checkbox).